### PR TITLE
Add RubocopMinitest

### PR DESCRIPTION
**Pull request by duduribeiro**
Hey folks 👋

I don't know if this commit is valid. 

We use rubocop-minitest and it would be great if codeclimate could support it.

This commit includes rubocop-minitest gem in the Gemfile to allow
users that uses it to have their code analyzed with CodeClimate.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -47,6 +47,7 @@ GEM
       unicode-display_width (~> 1.0, >= 1.0.1)
     rubocop-migrations (0.1.2)
       rubocop (~> 0.41)
+    rubocop-minitest (0.0.1)
```
Comment (andyw8) [inline Gemfile.lock]: The current release seems to be 0.2.1:

https://github.com/rubocop-hq/rubocop-minitest/releases

Comment (duduribeiro) [reply Gemfile.lock]: hey @andyw8 👋 

Thanks for reviewing. Yes, current release is 0.2.1, but to use > 2.0, rubocop core should be updated. Can I update it?

thanks 🍻

**Comment by andyw8** [comment]
I see, so that's causing it to resolve to an older version of rubocop-minitest.

There are 'channels' which use a newer version of RuboCop, e.g. https://github.com/codeclimate/codeclimate-rubocop/tree/channel/rubocop-0-74, so it might be better to create a PR against that branch.

**Comment by duduribeiro** [comment]
Awesome!.. thanks for your suggestion

**Comment by filipesperandio** [comment]
Sorry this is taking so long.
I am currently updating some channels to include latest rubocop version, if you don't mind doing this against the latest `channel/rubocop-0-76` branch I will be happy to merge this in.

**Comment by duduribeiro** [comment]
hey @filipesperandio 👋 

thanks for the comment. I made it against the channel/rubocop-0-76 branch. Thanks so much 🍻

File: Gemfile [hunk 2]
```diff
@@ -9,8 +9,10 @@ gem "pry", require: false
 gem "rubocop", "0.75.1", require: false
 gem "rubocop-performance", require: false
 gem "rubocop-migrations", require: false
-gem "rubocop-rspec", require: false
+gem "rubocop-minitest", require: false
+gem "rubocop-performance", require: false
```
Comment (filipesperandio) [inline Gemfile]: This got duplicated with the change.

Comment (duduribeiro) [reply Gemfile]: sorry for it.. pushed the fix. thanks!

**Pull request merged**